addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/InputWithDabBandControl.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yamahareceiver.internal.protocol;
+
+import com.google.common.collect.Sets;
+import org.openhab.binding.yamahareceiver.YamahaReceiverBindingConstants;
+
+import java.io.IOException;
+import java.util.Set;
+
+/**
+ * The DAB Band control protocol interface.
+ *
+ * @author Tomasz Maruszak - [yamaha] Tuner band selection and preset feature for dual band models (RX-S601D)
+ */
+public interface InputWithDabBandControl extends IStateUpdateable {
+    /**
+     * List all inputs that are compatible with this kind of control
+     */
+    Set<String> supportedInputs = Sets.newHashSet(YamahaReceiverBindingConstants.INPUT_TUNER);
```

Comment:
Efficient in performance or in writing in compact code? Because the runtime efficiency will not be hurt by this.

Maybe there should be an official opinion about the future of using Guave @sjka started with voting against and I agree with him that for this use case 820KB is rather heavy to drag along.

@kaikreuzer what is your idea about this?